front/app/modules/commercial/custom_topics/admin/components/TopicsList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Extracted this from the `front/app/modules/commercial/custom_topics/admin/containers/TopicsSettings/all/index.tsx` component, just to make the component a bit smaller and the code easier to read.